File: C/greatest_of_three.c
#include <stdio.h>
int main() {
	int a = 10, b = 20, c =30;
	int result;
    if (a >= b && a >= c) {
        result = a;
    } else if (b >= a && b >= c) {
        result = b;
    } else {
        result = c;
    }
    printf("greatest number is %d", result );
}

File: Python/greatest_of_three.py
def main():
    a = 10
    b = 20
    c = 30

    if a >= b and a >= c:
        result = a
    elif b >= a and b >= c:
        result = b
    else:
        result = c

    print(f"greatest number is {result}")


if __name__ == "__main__":
    main()

